src/geos.h

```diff
@@ -85,6 +86,9 @@ enum {
     case PGERR_EMPTY_GEOMETRY:                                                           \
       PyErr_SetString(PyExc_ValueError, "One of the Geometry inputs is empty.");         \
       break;                                                                             \
+    case PGWARN_GEOS_EXCEPTION:                                                          \
+      PyErr_WarnEx(PyExc_Warning, last_error, 0);                                        \
```

Comment:
We should probably prepend that last_error message to make the warning message clearer? (saying something about some values being invalid WKB and being replaced with None in the result)